can/common.pxd

```diff
@@ -9,6 +9,8 @@ from libcpp.vector cimport vector
 from libcpp.unordered_set cimport unordered_set
 
 
+ctypedef unsigned int (*calc_checksum_type)(uint32_t, const Signal&, const vector[uint8_t] &)
```

Comment:
Well that was easy, not sure how I got so turned around thinking it was harder. Thanks for the assist!